src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -104,7 +104,9 @@ protected Analyzer(SelectionResult selectionResult, CancellationToken cancellati
                 var typeParametersInDeclaration = GetMethodTypeParametersInDeclaration(returnType, sortedMap);
 
                 // check various error cases
-                var operationStatus = GetOperationStatus(model, symbolMap, parameters, unsafeAddressTakenUsed, returnTypeHasAnonymousType);
+                var operationStatus = dataFlowAnalysisData.Succeeded
+                    ? GetOperationStatus(model, symbolMap, parameters, unsafeAddressTakenUsed, returnTypeHasAnonymousType)
+                    : OperationStatus.FlowAnalysisFailed;
```

Comment:
this doesn't give user how to make extract method work. this error message seems same as not giving any message to user.